app/services/editions/EditionsTemplating.scala

```diff
@@ -5,51 +5,105 @@ import java.time.LocalDate
 import logging.Logging
 import model.editions._
 import play.api.mvc.{Result, Results}
-import services.editions.prefills.{Prefill, PrefillParamsAdapter}
+import services.editions.prefills.PrefillHelper.defineContentQueryTimeWindow
+import services.editions.prefills.{CapiQueryTimeWindow, Prefill, PrefillParamsAdapter}
 import services.{Capi, Ophan}
 
 import scala.concurrent.Await
 import scala.concurrent.duration._
 import scala.language.postfixOps
 import scala.util.control.NonFatal
 
+case class GenEditionTemplateResult(issueSkeleton: EditionsIssueSkeleton, contentPrefillTimeWindow: CapiQueryTimeWindow)
```

Comment:
"GenEditionTemplateResult" is not the clearest name.  What is it?

Reply:
yeah agree :)
The main idea is that this is a result of `generateEditionTemplate` function
where the skeleton is the actual data produced, additionally time-window is passed here as it is needed to persist that in DB

I can rename it to `GenerateEditionTemplateResult` ?

Reply:
done